scene/gui/item_list.cpp

```diff
@@ -1004,6 +1006,7 @@ void ItemList::_notification(int p_what) {
 		case NOTIFICATION_TRANSLATION_CHANGED:
 		case NOTIFICATION_THEME_CHANGED: {
```

Comment:
Probably better to split these cases to avoid unnecessary translation lookups on unrelated theme changes.

```c++
		case NOTIFICATION_TRANSLATION_CHANGED: {
 			for (int i = 0; i < items.size(); i++) {
				items.write[i].xl_text = atr(items[i].text);
				_shape_text(i);
			}
			shape_changed = true;
			queue_redraw();
		} break;
		case NOTIFICATION_THEME_CHANGED: {
 			for (int i = 0; i < items.size(); i++) {
				_shape_text(i);
			}
			shape_changed = true;
			queue_redraw();
		} break;
````

Reply:
Thanks for the feedback! I added the change you asked for :)